internal/entitlement/httpdriver/parser.go

```diff
@@ -28,7 +28,7 @@ func (parser) ToMetered(e *entitlement.Entitlement) (*api.EntitlementMetered, er
 		Id:              &metered.ID,
 		IsUnlimited:     convert.ToPointer(false), // implement
 		IssueAfterReset: metered.IssuesAfterReset,
-		Metadata:        nil, // implement
+		Metadata:        &metered.Metadata,
```

Comment:
Please make sure that we are not taking the address of a nil map

Reply:
ent populates that field in all cases and it always sets it to an empty map when there's no value (the field is not nillable), so it will not result in an error. do you wan't me to add safe dereferencing now?